plugin/core/open.py

```diff
@@ -15,6 +15,26 @@
 opening_files = {}  # type: Dict[str, Tuple[Promise[Optional[sublime.View]], ResolveFunc[Optional[sublime.View]]]]
 
 
+def _return_existing_view(flags: int, open_file_group: int, active_group: int, specified_group: int) -> bool:
+    open_side_by_side = bool(flags & (sublime.ADD_TO_SELECTION | sublime.REPLACE_MRU))
+    file_in_active_group = open_file_group == active_group
+
+    # true if DONT want to force it to open in the active group
+    # i.e. just jump to the file
+    select_file = not bool(flags & sublime.FORCE_GROUP)
+
+    if specified_group > -1:
+        return False
+    if open_side_by_side:
+        # default is -1... and is ignored
+        # always open in the specific group if pass in
+        return False
+    elif file_in_active_group:
+        return True
+    else:
+        return select_file
```

Comment:
I haven't looked at this logic-wise yet but would suggest reformatting like:
```suggestion
def _return_existing_view(flags: int, open_file_group: int, active_group: int, specified_group: int) -> bool:
    open_side_by_side = bool(flags & (sublime.ADD_TO_SELECTION | sublime.REPLACE_MRU))
    file_in_active_group = open_file_group == active_group
    if specified_group > -1:
        return False
    if open_side_by_side:
        # default is -1... and is ignored. always open in the specific group if pass in
        return False
    if file_in_active_group:
        return True
    # true if DON'T want to force it to open in the active group i.e. just jump to the file
    return not bool(flags & sublime.FORCE_GROUP)
```

- no need for `elif` after return
- no need to break short comments into mulitple lines
- we tend to not add unnecessary blank lines within function blocks